SharpBucket/V2/EndPoints/RepositoriesEndPoint.cs

```diff
@@ -22,24 +22,37 @@ public RepositoriesEndPoint(SharpBucketV2 sharpBucketV2)
         {
         }
 
+        
         /// <summary>
         /// List of repositories associated with an account. If the caller is properly authenticated and authorized, 
         /// this method returns a collection containing public and private repositories. 
         /// Otherwise, this method returns a collection of the public repositories. 
         /// </summary>
         /// <param name="accountName">The account whose repositories you wish to get.</param>
-        /// <param name="max">The maximum number of items to return. 0 returns all items.</param>
         /// <returns></returns>
-        public List<Repository> ListRepositories(string accountName, int max = 0)
+        public List<Repository> ListRepositories(string accountName) => ListRepositories(accountName, new ListParameters());
+
+        /// <summary>
+        /// List of repositories associated with an account. If the caller is properly authenticated and authorized, 
+        /// this method returns a collection containing public and private repositories. 
+        /// Otherwise, this method returns a collection of the public repositories. 
+        /// </summary>
+        /// <param name="accountName">The account whose repositories you wish to get.</param>
+        /// <param name="parameters">Parameters for the query.</param>
+        /// <returns></returns>
+        public List<Repository> ListRepositories(string accountName, ListParameters parameters)
         {
+            if (parameters == null)
+                throw new ArgumentNullException(nameof(parameters));
+
             var overrideUrl = _baseUrl + accountName + "/";
-            return GetPaginatedValues<Repository>(overrideUrl, max);
+            return GetPaginatedValues<Repository>(overrideUrl, parameters.Max, parameters.ToDictionary());
         }
 
         /// <summary>
         /// List of all the public repositories on Bitbucket.  This produces a paginated response. 
         /// Pagination only goes forward (it's not possible to navigate to previous pages) and navigation is done by following the URL for the next page.
-        /// The returned repositories are ordered by creation date, oldest repositories first. Only public repositories are returned.
+        /// The returned repositories are ordered by creation date, oldest repositories first. Only public repositories are returned.      
```

Comment:
Useless spaces added at the end of the line
```suggestion
        /// The returned repositories are ordered by creation date, oldest repositories first. Only public repositories are returned.
```